### DISCUSSION TITLE
Possible to change the <Collapse> components 'title' prop size?

### DISCUSSION THREAD

**Discussion by gordod26** [zeit-ui/react#557] [Q&A] [answered]
Hi, 

As the title says, can someone help me find a way to change the size of the Collapse components 'title' prop?

**Comment by ghost** [comment, marked as answer]
In case of JavaScript you could pass a JSX element in `title`. You probably can't change the size of `title` in TypeScript as it has a string only type.

![image](https://user-images.githubusercontent.com/66206863/121810936-40349480-cc80-11eb-8b33-d934c6907192.png)

**Reply by unix** [reply]
Yes, this can be improved, the title should support both string and element modes.

**Reply by gordod26** [reply]
Thank you! I got it to work.
